softvis3d-frontend/src/constants/Metrics.ts

```diff
@@ -42,12 +42,16 @@ export const coverageColorMetric: Metric = new Metric(coverageMetricId, "Coverag
 export const openIssuesColorMetric: Metric = new Metric("open_issues", "Open Issues",
     "Scale from green (no open issues) to red (a lot of issues).");
 
+export const numberOfAuthorsBlameColorMetric: Metric = new Metric("number_of_authors_blame",
```

Comment:
Do we need the "(blame)" part? Seems kind of unnecessary...

Also it seems to emphasize this metric in comparison to the others.